src/main/java/com/iota/iri/zmq/MessageQ.java

```diff
@@ -51,7 +52,7 @@ public void shutdown() {
             LOG.error("Publisher service shutdown failed.", e);
         }
 
-        publisher.close();
-        context.term();
+        IOUtils.closeQuietly(publisher);
+        IOUtils.closeQuietly(context);
```

Comment:
Context implements `Closeable`, and `close()` in practice calls `term()`:
https://github.com/zeromq/jzmq/blob/master/jzmq-jni/src/main/java/org/zeromq/ZMQ.java#L512